src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -766,14 +766,14 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                             $maxDepth = 10
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-macos`
Tests for $ErrorView.ConciseView tests.Error shows if $PSModuleAutoLoadingPreference is set to 'none'
```powershell
Expected like wildcard '*cmdletThatDoesntExist*' to match <empty>, but it did not match.
at <ScriptBlock>, /Users/runner/runners/2.168.2/work/1/s/test/powershell/engine/Formatting/ErrorView.Tests.ps1: line 111
111:             $e | Should -BeLike "*cmdletThatDoesntExist*"
```